src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,114 @@
+package wolox.training.models;
+
+import com.sun.istack.NotNull;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "book")
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    private String genre;
+    @NotNull
+    private String author;
```

Comment:
The @Column annotations are required to be able to generate the model in the database.
In this case I would add the @Column (nulleable = false) annotation in conjunction with the @NotNull annotation(this should in the rest of the fields except for the id).

```suggestion
    @NotNull
    @Column (nulleable = false)
    private String author;
```

Reply:
Add Column annotation again